judge/views/two_factor.py

```diff
@@ -225,10 +226,11 @@ def post(self, request, *args, **kwargs):
         return HttpResponse()
 
 
-class TwoFactorLoginView(SuccessURLAllowedHostsMixin, TOTPView):
+class TwoFactorLoginView(SuccessURLAllowedHostsMixin, TOTPView, ContextMixin):
     form_class = TwoFactorLoginForm
     title = gettext_lazy('Perform Two-factor Authentication')
     template_name = 'registration/two_factor_auth.html'
+    extra_context = {'twofa_active': True}
```

Comment:
Please don't call it `twofa_active`. `twofa` hurts my eyes. Call it `_2fa` or `tfa` or `mfa` or anything else. Also it's not `active`, it's really "in progress".

Reply:
Renamed